apps/io-services-cms-webapp/host.json

```
@@ -38,5 +38,17 @@
   "extensionBundle": {
     "id": "Microsoft.Azure.Functions.ExtensionBundle",
     "version": "[4.*, 5.0.0)"
-  }
+  },
+  "functions": [
+    "GetServices",
+    "GetServiceLifecycle",
+    "GetServicePublication",
+    "GetServiceHistory",
+    "CreateService",
+    "EditService",
+    "DeleteService",
+    "PublishService",
+    "UnPublishService",
+    "ReviewService"
+  ]
```

chore: I think this is an oversight 😉 
```suggestion
  }
```

---

yup, pushed by mistake, removed

---

a0482a604321c423e215cc545208d89e2095079d